apps/system/js/sim_lock_manager.js

```diff
@@ -159,6 +159,7 @@
 
     '_handle_airplanemode-enabled': function(evt) {
       this.simLockSystemDialog.close();
+      this._alreadyShown = false;
```

Comment:
Don't need to fix in this bug: I hope we could use this.simLockDialog.isVisible() instead of keeping and maintaining this internal value in the future. It might be async with the real state and cause confusion - unless there is an obvious reason to do so.